vagrant/install-ruby.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+source $HOME/.rvm/scripts/rvm
+
+rvm use --default --install $1
+echo "gem: --no-rdoc --no-ri" > ~/.gemrc

```

Comment:
Add these comment:
# disable gem documentation install to speed up gem installation